src/utilities/omit.js

```diff
@@ -0,0 +1,5 @@
+export default function omit(object = {}, ...keys) {
+  const copy = Object.assign({}, object);
+  keys.forEach(key => delete copy[key]);
+  return copy;
+}
```

Comment:
> ### 5.2.1. Objects that are in "hash table mode" (aka "normalized objects", "dictionary mode" - objects who have a hash table as a backing data structure) are not simple enumerables**
> 
> ```js
> function hashTableIteration() {
>     var hashTable = {"-": 3};
>     for(var key in hashTable);
> }
> ```
>
> An object will go into hash table mode for example when you add too many properties dynamically (outside constructor), delete properties, use properties that cannot be valid identifiers and so on. In other words, when you use an object as if it was a hash table, it will be turned into a hash table. Passing such an object to for-in is a no no. You can tell if an object is in hash table mode by calling console.log(%HasFastProperties(obj)) when the flag --allow-natives-syntax is enabled in Node.JS.
> — ["Optimization killers"](https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#521-objects-that-are-in-hash-table-mode-aka-normalized-objects-dictionary-mode---objects-who-have-a-hash-table-as-a-backing-data-structure-are-not-simple-enumerables) by petkaantonov (bluebird's author, and bluebird is fastest promises implementation)